docs/introduction/partials/_getting-started-rn.md

```diff
@@ -1,16 +1,15 @@
 <!-- markdownlint-configure-file { "first-line-h1": 0 } -->
 
+Detox was built from the ground up to support "classic" React Native projects.
+
 :::info Note
 
-Mind that Detox works, at least officially, with the following React Native versions:
+While Detox should work out of the box with almost any React Native version from the past few minor releases, official support is provided for React Native versions `0.70.x` and `0.71.x` without React Native's ["New Architecture"](https://reactnative.dev/docs/the-new-architecture/landing-page).
 
-| iOS             | Android                                                                                                                |
-| --------------- | ---------------------------------------------------------------------------------------------------------------------- |
-| 0.68.x - 0.70.7 | 0.68.x - 0.70.7 -<br/>Visibility edge-case: see this [RN issue](https://github.com/facebook/react-native/issues/23870) |
+Official support means that we test Detox against these versions and potentially fix issues that are specific to them. This does not mean that Detox will not work with other versions, but rather that we do not test Detox against them and do not fix issues that are specific to them.
 
-Newer versions, as well as React Native's ["New Architecture"](https://reactnative.dev/docs/the-new-architecture/landing-page), are most likely supported, but have not been tested out yet.
-Please [submit an issue](https://github.com/wix/Detox/issues/new/choose) if you find specific problems with more recent React Native versions.
+Newer versions, as well as React Native's "New Architecture", may work with Detox, but have not been tested out yet by the Detox team. In case of a problem with a more recent React Native version or with React Native's "New Architecture", please [submit an issue](https://github.com/wix/Detox/issues/new/choose) and we will do our best to help you out and to address the issue once we will support these versions officially.
 
-For support for React Native 0.71.x, please track [this issue](https://github.com/wix/Detox/issues/3884).
+Although we do not officially support older React Native versions, we do our best to keep Detox compatible with them to the extent possible, as long as it does not require us to compromise on the quality of Detox's codebase.
```

Comment:
```suggestion
Also, in case of a problem with an unsupported version of React Native, please [submit an issue](https://github.com/wix/Detox/issues/new/choose) or write us in our [Discord server](https://discord.gg/CkD5QKheF5) and we will do our best to help out.
```